src/features/groups/components/ContainerList/CreateContainerButton/CreateContainerBottomDrawerContent.tsx

```diff
@@ -0,0 +1,95 @@
+import {
+  Button,
+  DrawerBody,
+  DrawerClose,
+  DrawerContent,
+  DrawerFooter,
+  DrawerHeader,
+  DrawerTitle,
+  Input,
+} from '@/components/ui';
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from '@/components/ui/Form';
+import { createContainerFormSchema, CreateContainerInputs } from '@/features/groups/lib/schemas';
+
+import { zodResolver } from '@hookform/resolvers/zod';
+import { FC, useEffect } from 'react';
+import { SubmitHandler, useForm } from 'react-hook-form';
+import { z } from 'zod';
+
+interface ICreateContainerDrawerContentProps {
+  isOpen: boolean;
+  onClose: () => void;
+}
```

Comment:
Could you please also add TSDoc to each property of this interface? There are many existing interfaces that still do not have TSDoc, though. It's a best practice to always add TSDocs to interfaces, as well as functions.

You can get an idea by visiting the `src/components/layouts/MobileLayout.tsx` file.
Also this PR (https://github.com/nishiki-tech/nishiki-frontend/pull/246/files) includes a lot of TSDocs for interfaces.